drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/EdgeDriverFactory.java

```diff
@@ -60,8 +61,9 @@ protected String getDriverReadableName() {
 
     public Capabilities getCapabilities(WebDriverConfiguration configuration, boolean performValidations) {
         DesiredCapabilities capabilities = new DesiredCapabilities(configuration.getCapabilities());
-        capabilities.setPlatform(DesiredCapabilities.edge().getPlatform());
+        capabilities.setPlatform(Platform.ANY);
         capabilities.setBrowserName(DesiredCapabilities.edge().getBrowserName());
+//        capabilities.setCapability("useChromium", "true");
```

Comment:
Commented line

Reply:
Right :) That's when I was testing Edge for Linux. You could use suggestions here to remove:

```suggestion
```